MANIFEST.in

```diff
@@ -1,4 +1,4 @@
 include requirements.txt
 include README.rst
 include CHANGELOG.rst
-include LICENSE
```

Comment:
no

Reply:
There was a missing empty line here, hence GitHub with the red symbol.  This standardizes all files to have one empty line at the end

Reply:
My point was, please do not touch the file endings, all files are not equal.

Reply:
Multiple companies have multiple policies and standards....we are lucky we only have our own :-)

I will review your PR again, when CI is green, but you have to find better arguments that "multiple....." to convince me that adding end-of-file-fixer is worth it.

Reply:
Haha yeah sure.  CI is green, if you would like me to remove `end-of-file-fixer`, lemme know

Reply:
I do not want that check, at least not in this PR. If we want that check (and it is a big if) it should be done "loudly" in its own PR.